ui/app/templates/vault/cluster/policies/create.hbs

```diff
@@ -46,7 +46,7 @@
         </div>
       </div>
       {{#if showFileUpload}}
-        <TextFile @inputOnly={{true}} @index="" @file={{file}} @onChange={{action "setPolicyFromFile"}} />
+        <TextFile @inputOnly={{true}} @file={{file}} @onChange={{action "setPolicyFromFile"}} />
```

Comment:
index was one of those paramaters in TextFile that was defaulted to null and always passed in as empty string.  Changed so that it is always an empty string.